### PR TITLE
[Brl.Clipboard] Added docs

### PR DESCRIPTION
**Pull request by GWRon**
The PR contains docs for the procural functions of the pull: https://github.com/bmx-ng/brl.mod/pull/124

So only merge this if you intend to merge the other PR.

### CODE REVIEW

File: clipboard.mod/doc/intro.bbdoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
++Clipboard
```
Comment (woollybah) [inline clipboard.mod/doc/intro.bbdoc]: Probably don't need this, as the page will already have a title.

**Comment by GWRon** [comment]
Did so as intro.bbdoc of linkedlist contained it too.

**Comment by woollybah** [comment]
Doesn't mean it's right though ;-)

**Comment by GWRon** [comment]
Will create a new PR without that line...once I am on my computer.

And another one to remove that line from TList.

**Pull request closed without merging**